package.json

```diff
@@ -141,5 +143,8 @@
   },
   "optionalDependencies": {
     "fsevents": "*"
+  },
+  "jest": {
+    "testURL": "http://localhost"
```

Comment:
Can remove this once this [patch](https://github.com/facebook/jest/pull/6792) is published. JSDom (used underneath Jest) made a change upstream and the internet is losing its mind.

Reply:
thx much